core/modules/utils/dom/animations/slide.js

```diff
@@ -80,6 +79,7 @@ function slideClosed(domNode,options) {
 		if(options.callback) {
 			options.callback();
 		}
+		domNode.style.removeProperty('transition');
```

Comment:
slide is an `animation` and not a storyview. I do not know, how to test this one. But I think it should be part of a different PR - except it is broken and we get a description how to test it.